assets/src/stories-editor/helpers/index.js

```diff
@@ -502,9 +494,28 @@ export const wrapBlocksInGridLayer = ( element, blockType, attributes ) => {
 		};
 	}
 
+	const animationAtts = {};
+
+	// Add animation if necessary.
+	if ( ampAnimationType ) {
+		animationAtts[ 'animate-in' ] = ampAnimationType;
```

Comment:
This doesn't seem to work reliably.

Everything is fine when editing a block and previewing it on the front end, but upon refreshing the page I get block validation errors because of the missing attributes:

![Screenshot 2019-05-27 at 15 48 23](https://user-images.githubusercontent.com/841956/58424182-2b840480-8097-11e9-8cb4-18b5f84cd379.png)

Reply:
Ugh, I had forgotten to update the attribute selectors. Should work now.